### PR TITLE
Renaming ethereum endpoints

### PR DESCRIPTION
**Pull request by darioAnongba**
Rename `eth` to `ethereum` in endpoints and use `eth` instead of `eth1` internally as naming

### CODE REVIEW

**Comment by dolanor** [comment]
Why the eth1 name was selected? In case we would support eth2 as well? Is there a plan to have some eth2 quorum?

File: deps/ca/ca.srl [hunk 1]
```diff
@@ -0,0 +1 @@
+20EBCB11194D5794BB66DBC343B86D9D15BA47EB
```
Comment (ggarri) [inline deps/ca/ca.srl:1]: I thought you decied on removing them. Why did you add it again?

File: deps/config/apikey/sample.csv [hunk 2]
```diff
@@ -2,6 +2,6 @@
 ## Column separator is ";"
 
 # sha256("admin-user")
-f470213d9ae659187a19b9cb2169b4b400544f4d3f59250eda657154700da616;tenant1:user1;*:*;admin
+f470213d9ae659187a19b9cb2169b4b400544f4d3f59250eda657154700da616;tenant1|user1;*:*;admin
 # sha256("guest-user")
-8605f70ff5f55e2a9323d97de3dbf8e61f38314d93298ca00e19f8918fe8971b;tenant2:user2;*:secrets,*:keys,read:eth1accounts;
+8605f70ff5f55e2a9323d97de3dbf8e61f38314d93298ca00e19f8918fe8971b;tenant2|user2;*:secrets,*:keys,read:ethaccounts;anonymous
```
Comment (ggarri) [inline deps/config/apikey/sample.csv:7]: What was the final resource name? `ethereum` or `ethaccount` ? In the template you are using `ethereum` and here `ethaccount`

File: go.mod [hunk 3]
```diff
@@ -30,14 +30,14 @@ require (
...
 	github.com/justinas/alice v1.2.0
+	github.com/lib/pq v1.8.0
 	github.com/magefile/mage v1.10.0 // indirect
 	github.com/oxtoacart/bpool v0.0.0-20190530202638-03653db5a59c
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
-	github.com/swaggo/swag v1.7.1
```
Comment (ggarri) [inline go.mod:40]: We keep going back and forth with this? this you try to run `$> make install-swag` already?

File: deps/ca/ca.srl (hunk 1 above)
Comment (darioAnongba) [reply deps/ca/ca.srl:1]: mistake

File: deps/config/manifests/roles.yml.template [hunk 4]
```diff
@@ -14,7 +14,7 @@
     permission:
       - "read:*"
       - "sign:keys"
-      - "sign:eth1accounts"
+      - "sign:ethereum"
```
Comment (ggarri) [inline deps/config/manifests/roles.yml.template:17]: I think this should be `sign:ethaccount`

File: src/auth/authenticator/tls/authenticator_test.go [hunk 5]
```diff
@@ -57,7 +57,7 @@ func TestAuthenticatorSameCert(t *testing.T) {
 		assert.Equal(t, "eve", userInfo.Username)
 		assert.Equal(t, "auth0", userInfo.Tenant)
 		assert.Equal(t, []string{"signer"}, userInfo.Roles)
-		assert.Equal(t, types.ListWildcardPermission("*:eth1accounts"), userInfo.Permissions)
+		assert.Equal(t, types.ListWildcardPermission("*:ethereum"), userInfo.Permissions)
```
Comment (ggarri) [inline src/auth/authenticator/tls/authenticator_test.go:60]: Again, I am confused. Should it be `ethaccounts` ?

File: deps/ca/ca.srl (hunk 1 above)
Comment (darioAnongba) [reply deps/ca/ca.srl:1]: removed

File: deps/config/apikey/sample.csv (hunk 2 above)
Comment (darioAnongba) [reply deps/config/apikey/sample.csv:7]: mistake

Comment (darioAnongba) [reply deps/config/apikey/sample.csv]: changed

File: go.mod (hunk 3 above)
Comment (darioAnongba) [reply go.mod:40]: Is it okay now? I ran the command and then `go mod tidy`

File: src/auth/authenticator/tls/authenticator_test.go (hunk 5 above)
Comment (darioAnongba) [reply src/auth/authenticator/tls/authenticator_test.go:60]: why? `ethereum` is clearly a better name than `ethaccounts` since the domain is named `ethereum` so it correlates with keys and secrets

File: deps/config/manifests/roles.yml.template (hunk 4 above)
Comment (darioAnongba) [reply deps/config/manifests/roles.yml.template:17]: why? `ethereum` is clearly a better name than `ethaccounts` since the domain is named `ethereum` so it correlates with keys and secrets

File: src/auth/manager/service_test.go [hunk 6]
```diff
@@ -32,19 +32,19 @@ var testManifest = []byte(`
   name: signer
   specs:
     permission:
-      - read:eth1accounts
+      - read:ethaccounts
```
Comment (ggarri) [inline src/auth/manager/service_test.go:35]: I guess this should be renamed to `read:ethereum`, right?

File: deps/config/manifests/store.yml.template [hunk 7]
```diff
@@ -58,9 +58,9 @@
       address: http://hashicorp:8200
       token: '{VAULT_TOKEN}'
       namespace: ''
-- kind: Eth1Account
+- kind: EthAccount
```
Comment (ggarri) [inline deps/config/manifests/store.yml.template:61]: I would rather rename this to "Ethereum" to correlate with the rest of the changes

File: src/auth/manager/service_test.go (hunk 6 above)
Comment (darioAnongba) [reply src/auth/manager/service_test.go:35]: lol why are these tests passing. Updated

File: deps/config/manifests/store.yml.template (hunk 7 above)
Comment (darioAnongba) [reply deps/config/manifests/store.yml.template]: applied

**Review by ggarri — APPROVED** [review]
